extra/Dockerfile

```diff
@@ -8,12 +8,15 @@ ARG RUN_GID=0
 COPY . /tmp/motioneye
 
 RUN case "$(dpkg --print-architecture)" in \
-      'armhf') PACKAGES=''; printf '%b' '[global]\nextra-index-url=https://www.piwheels.org/simple/\n' > /etc/pip.conf;; \
+      'armhf') PACKAGES='python3-distutils'; printf '%b' '[global]\nextra-index-url=https://www.piwheels.org/simple/\n' > /etc/pip.conf;; \
       *) PACKAGES='python3-dev libcurl4-openssl-dev gcc libssl-dev';; \
     esac && \
-    apt-get update && \
-    DEBIAN_FRONTEND="noninteractive" apt-get -y --option Dpkg::Options::="--force-confnew" --no-install-recommends install \
-      python3 python3-pip $PACKAGES tzdata && \
+    apt-get -q update && \
+    DEBIAN_FRONTEND="noninteractive" apt-get -qq --option Dpkg::Options::="--force-confnew" --no-install-recommends dist-upgrade && \
```

Comment:
I wouldn't recommend updating the base os inside of a docker container. The base images are always up to date, just if there where a fresh install. The background is that Docker images are stored in layers. Layers are stored as differences to the previews layer. Every instruction in a Dockerfile creates a new layer. If you use the base os and in a layer, after that, you do a full upgrade of the os, you basically shipping the os two times. In sense of space.

Since we are using `debian:bullseye-slim` and not a fixed sha265 or a specific major minor patch version, we always get the latest version of debian bullseye.

---
Just an idea, nothing for this pr:

We also could use a multistage build with python as base images. There we could compile the python packages in the first stage and copy the compiled artifacts to the second stage, where we also don't have to install the devel packages of OpenSSL or curl and just the lib packages.

https://github.com/PyDrocsid/MorpheusHelper/blob/932a2ebeb3ba97e5ee31f02dd8e89cd64bbf699a/Dockerfile
With this approach, we could compile everything ourselves, and are able to specify the version for every ourself and don't depend on the versions available in the os package sources.
I think we would easily be able to get down to 500mb (currently about 700mb)

Reply:
> I wouldn't recommend updating the base os inside of a docker container.

CodeFactor also shows an annotation about this. Shall we keep the `apt-get -qq upgrade` we previously had in place or skip this as well?

Reply:
yes, an upgrade also shouldn't be done.
systems isn't in a default official base image

I am going to build a dockerfile as I described in the next few days/weeks.

Reply:
> yes, an upgrade also shouldn't be done.

Updating it.

Ah right, `systemd` is not present indeed, we actively skip it with `motioneye_init` in the Dockerfile.

> I am going to build a dockerfile as I described in the next few days/weeks.

🚀